components/kyma-environment-broker/internal/process/upgrade_kyma/cls_binding.go

```diff
@@ -131,7 +126,7 @@ func (s *ClsUpgradeBindStep) Run(operation internal.UpgradeKymaOperation, log lo
 		}
 	}
 
-	operation.InputCreator.SetLabel(kibanaURLLabelKey, overrideParams.KibanaUrl)
+	//operation.InputCreator.SetLabel(kibanaURLLabelKey, overrideParams.KibanaUrl)
```

Comment:
do we need this kibana url?

Reply:
I checked with Benjamin and figured that they are not using those labels anymore. Will delete that field completely

Reply:
Fixed in 0e36826